kubernetes_deploy/api-sandbox/prometheus-custom-rules.yaml

```diff
@@ -14,30 +14,30 @@ spec:
       expr: 100 * kube_resourcequota{job="kube-state-metrics",type="used",namespace="cccd-api-sandbox"} / ignoring(instance, job, type) (kube_resourcequota{job="kube-state-metrics",type="hard",namespace="cccd-api-sandbox"} > 0) > 90
       for: 1m
       labels:
-        severity: laa-court-get-paid
+        severity: laa-cccd-alerts
```

Comment:
The previous version was `laa-court-get-paid` but this channel does not exist. Does `court` have some special purpose?

Reply:
the name is not related to the channel. it relates to a slack webhook and its "receiver" that CP create and give a name. I have just used the convention of naming the webhook after the channel, which CP have then used for the custom severity level. 

Its all a bit disconnected but we do no have insight or access to their creation of the "receiver".